Exiled.Events/Handlers/Map.cs

```diff
@@ -147,5 +154,11 @@ public static class Map
         /// Called after the map is generated.
         /// </summary>
         public static void OnGenerated() => Generated.InvokeSafely();
+
+        /// <summary>
+        /// Called before the server changes a <see cref="Pickup"/> into a <see cref="Grenade"/> when hit by an explosion.
+        /// </summary>
+        /// <param name="ev">The <see cref="ChangingIntoGrenadeEventArgs"/> instance.</param>
+        public static void OnServerChangingGrenade(ChangingIntoGrenadeEventArgs ev) => ChangingIntoGrenade.InvokeSafely(ev);
```

Comment:
It should be named `OnChangingIntoGrenade`.

Reply:
oop